test/utils_for_testing.cpp

```diff
@@ -547,6 +547,9 @@ void PnetIntegrationTestBase::cfg_init()
 
    /* Storage */
    strcpy (pnet_default_cfg.file_directory, "/disk1");
+
+   /* Diagnosis */
+   pnet_default_cfg.use_qualified_diagnosis = true;
```

Comment:
I can't see that this config is set for the sample application. I think it should.

Reply:
Done